src/content/en/2019/page-weight.md

```diff
@@ -6,809 +6,765 @@ description: Page Weight chapter of the 2019 Web Almanac covering why page weigh
 
-Unlike some of the other tables in this data set, this one has mostly happy takeaways. Videos are consistently smaller on mobile - which is great to see. In addition, the median size of an MP4 video is a very reasonable 18 KB on mobile and 39 KB of desktop. The median numbers for WebM are even better but they should be taken with a grain of salt: the duplicate measurement of “0.29 KB” across multiple clients and percentiles is a little bit suspicious. One possible explanation is that identical copies of one very tiny WebM video is included on many pages. Of the three formats, MPEG-TS consistently has the highest file size across all percentiles; this may be related to the fact that it was released in 1995 - making it the oldest of these three media formats.
-
-#### Mobile
-
-<table>
-  <tr>
-   <td><strong>Percentile</strong>
-   </td>
-   <td><strong>MP4 (KB)</strong>
-   </td>
-   <td><strong>WebM (KB)</strong>
-   </td>
-   <td><strong>MPEG-TS (KB)</strong>
-   </td>
-  </tr>
-  <tr>
-    <td>10</td>
-    <td>0.89</td>
-    <td>0.29</td>
-    <td>0.01</td>
-  </tr>
-  <tr>
-    <td>25</td>
-    <td>2.07</td>
-    <td>0.29</td>
-    <td>55</td>
-  </tr>
-  <tr>
-    <td>50</td>
-    <td>18</td>
-    <td>1.44</td>
-    <td>153</td>
-  </tr>
-  <tr>
-    <td>75</td>
-    <td>202</td>
-    <td>223</td>
-    <td>278</td>
-  </tr>
-  <tr>
-    <td>90</td>
-    <td>928</td>
-    <td>390</td>
-    <td>475</td>
-  </tr>
-</table>
-
-#### Desktop
-
-<table>
-  <tr>
-   <td><strong>Percentile</strong>
-   </td>
-   <td><strong>MP4 (KB)</strong>
-   </td>
-   <td><strong>WebM (KB)</strong>
-   </td>
-   <td><strong>MPEG-TS (KB)</strong>
-   </td>
-  </tr>
-  <tr>
-    <td>10</td>
-    <td>0.27</td>
-    <td>0.29</td>
-    <td>34</td>
-  </tr>
-  <tr>
-    <td>25</td>
-    <td>1.05</td>
-    <td>0.29</td>
-    <td>121</td>
-  </tr>
-  <tr>
-    <td>50</td>
-    <td>39</td>
-    <td>17</td>
-    <td>286</td>
-  </tr>
-  <tr>
-    <td>75</td>
-    <td>514</td>
-    <td>288</td>
-    <td>476</td>
-  </tr>
-  <tr>
-    <td>90</td>
-    <td>2142</td>
-    <td>896</td>
-    <td>756</td>
-  </tr>
-</table>
+Unlike some of the other tables in this data set, this one has mostly happy takeaways. Videos are consistently smaller on mobile—which is great to see. In addition, the median size of an MP4 video is a very reasonable 18 KB on mobile and 39 KB on desktop. The median numbers for WebM are even better but they should be taken with a grain of salt: the duplicate measurement of "0.29 KB" across multiple clients and percentiles is a little bit suspicious. One possible explanation is that identical copies of one very tiny WebM video is included on many pages. Of the three formats, MPEG-TS consistently has the highest file size across all percentiles; this may be related to the fact that it was released in 1995—making it the oldest of these three media formats.
```

Comment:
```suggestion
Unlike some of the other tables in this data set, this one has mostly happy takeaways. Videos are consistently smaller on mobile, which is great to see. In addition, the median size of an MP4 video is a very reasonable 18 KB on mobile and 39 KB on desktop. The median numbers for WebM are even better but they should be taken with a grain of salt: the duplicate measurement of 0.29 KB across multiple clients and percentiles is a little bit suspicious. One possible explanation is that identical copies of one very tiny WebM video is included on many pages. Of the three formats, MPEG-TS consistently has the highest file size across all percentiles. This may be related to the fact that it was released in 1995, making it the oldest of these three media formats.
```